.github/workflows/lint-and-test-frontend.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint and test - frontend
```

Comment:
I think it is a good idea to run only pipelines for the code that actually has changed. I have found this [repo](https://github.com/dorny/paths-filter#conditional-execution). Here is a section `Execute job in a workflow only if some file in a subfolder is changed` which has `backend` and `frontend` folders. Without these filters, it will be only wasting of resources to run tests for already working code. What do you think about that?

Reply:
This can be done natively:
![Screenshot_20210915-085602_Brave.jpg](https://user-images.githubusercontent.com/44314946/133385397-a0a95a78-ca5b-4f72-b12d-594b389b6479.jpg)
I have done it for pull requests as it only runs based on changes in files which have correct extension.
Do You think this should also be for push event?

Reply:
@Sojusan I thought about it a bit and changed this so the backend pipeline is run when any `.cs` or `.csproj` files are changed and frontend one is run when any `.ts` or `.tsx` is changed. The pipeline which pushes backend to container registry is run when there are any changes to files in the backend folder (but only on push, not on PR). If this satisfies You then please resolve this thread 😄